### ISSUE TITLE
"This version of RE4 is not supported" with Feb 27, 2023 update

### ISSUE THREAD

**Issue by ShadowOne333** [nipkownix/re4_tweaks#473]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.1: https://www.re4hd.com/?p=9552)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.1 SFV: [BIO4-HDProject1.1-SFV.zip](https://github.com/emoose/re4-research/files/9207370/BIO4-HDProject1.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

Good day.

I was playing RE4 HD just fine today, playing on Normal mode just finishing up Chapter 3-3. I started the chapter normally, and then attempted going for the shortcut that leads from the cage room to the room with the Broken Butterfly in it, and when trying to enter said room, I got kicked out of the game to what seemed to be an update of sorts for RE4 (around 500 or so MB)

I let the game update, and when attempting to play it again, I got the message "This version of RE4 is not supported. re4_tweaks will be disabled".

I have not edited nor modified any of the files, and I can confirm I am using version 1.8 of tweaks. I am not sure if this is a known issue as of yet or not.

**Comment by emoose** [comment]
Any chance you could upload your bio4.exe file somewhere? Looking at the EXEs I got from the update it seemed it was just the signature on the EXE changed, actual code inside looked to be identical for me, so should still work fine...

They do have a really weird depot setup on Steam that means you get different EXE depending on your region though, maybe the one you updated to was actually changed in some way.

**Comment by ShadowOne333** [comment]
Sure, here you go, let me know if you need anything else:
[bio4.zip](https://github.com/nipkownix/re4_tweaks/files/10847832/bio4.zip)

**Comment by JoelTroch** [comment]
> no idea whether any graphics/data was changed though.

According to SteamDB, only the EXE was changed and the assets were left untouched (example with the "WorldWide" depot here: https://steamdb.info/depot/254701/history/ The same treatment has been applied to the "German" and "Japanese" depots as well).

**Comment by ShadowOne333** [comment]
Sorry for the delayed response. I had to redownload the HD project again since I didn't have it handy, and the speed was around 2.4 MB/s, and add to that the 10~GB of RE4 and that was a good 4-5 hours.

Okay, so I did what emoose suggested, I went to the beta settings and indeed, I had 1.0.2 enabled for some reason, I changed this back to "None" and RE4 started downloading again.

After it finished, I ran the HD Project installation again from scratch, let it finish up, tested the game on Steam and it seems to be working fine now.

Thank you for the help, and sorry for wrongly thinking the update might have been the cause. Indeed, what Joel mentions is true, only the bio4.exe was updated, everything else is intact.

I will close this issue now, thanks again! And hope this helped in some way for others.
